lib/managers/FooManager.js

```diff
@@ -1,7 +1,6 @@
 const FooRepo = require('../repos/FooRepo');
 const BarServiceClient = require('../clients/BarServiceClient');
 const FooModel = require('../models/FooModel');
```

Comment:
I know this is not a change in this PR but maybe, if we want to have this repo as a goto we can update this to " instead of '?

Reply:
ah yes!

Reply:
fixed!